src/ConnectionManager.js

```diff
@@ -168,4 +196,26 @@ export default class ConnectionManager {
 
     this.next();
   }
+
+  /**
+   * Immediatly opens a given connection, cancels other connections if needed
+   *
+   * @this ConnectionManager~Context
+   * @param {AbstractConnection} connection
+   */
+  force(connection) {
```

Comment:
tests are missing for `force`, also I dont like the name… but `cut` (as in "cutting in a line") isnt very good - any ideas?